gen/xbuild.sh

```diff
@@ -8,51 +8,75 @@
-CGO_ENABLED=0 GOOS=openbsd   GOARCH=386      go build -ldflags="-s -w -X main.gVersion=$version" && tar czf dist/lf-openbsd-386.tar.gz     lf --remove-files
-CGO_ENABLED=0 GOOS=openbsd   GOARCH=amd64    go build -ldflags="-s -w -X main.gVersion=$version" && tar czf dist/lf-openbsd-amd64.tar.gz   lf --remove-files
-CGO_ENABLED=0 GOOS=openbsd   GOARCH=arm      go build -ldflags="-s -w -X main.gVersion=$version" && tar czf dist/lf-openbsd-arm.tar.gz     lf --remove-files
-CGO_ENABLED=0 GOOS=openbsd   GOARCH=arm64    go build -ldflags="-s -w -X main.gVersion=$version" && tar czf dist/lf-openbsd-arm64.tar.gz   lf --remove-files
-CGO_ENABLED=0 GOOS=solaris   GOARCH=amd64    go build -ldflags="-s -w -X main.gVersion=$version" && tar czf dist/lf-solaris-amd64.tar.gz   lf --remove-files
+ERRORS=
 
-CGO_ENABLED=0 GOOS=windows   GOARCH=386      go build -ldflags="-s -w -X main.gVersion=$version" && zip dist/lf-windows-386.zip            lf.exe --move
-CGO_ENABLED=0 GOOS=windows   GOARCH=amd64    go build -ldflags="-s -w -X main.gVersion=$version" && zip dist/lf-windows-amd64.zip          lf.exe --move
+for params in \
+    "android arm64 lf-android-arm64.tar.gz lf" \
+    "darwin amd64 lf-darwin-amd64.tar.gz lf" \
+    "dragonfly amd64 lf-dragonfly-amd64.tar.gz lf" \
+    "freebsd 386 lf-freebsd-386.tar.gz lf" \
+    "freebsd amd64 lf-freebsd-amd64.tar.gz lf" \
+    "freebsd arm lf-freebsd-arm.tar.gz lf" \
+    "illumos amd64 lf-illumos-amd64.tar.gz lf" \
+    "linux 386 lf-linux-386.tar.gz lf" \
+    "linux amd64 lf-linux-amd64.tar.gz lf" \
+    "linux arm lf-linux-arm.tar.gz lf" \
+    "linux arm64 lf-linux-arm64.tar.gz lf" \
+    "linux ppc64 lf-linux-ppc64.tar.gz lf" \
+    "linux ppc64le lf-linux-ppc64le.tar.gz lf" \
+    "linux mips lf-linux-mips.tar.gz lf" \
+    "linux mipsle lf-linux-mipsle.tar.gz lf" \
+    "linux mips64 lf-linux-mips64.tar.gz lf" \
+    "linux mips64le lf-linux-mips64le.tar.gz lf" \
+    "linux s390x lf-linux-s390x.tar.gz lf" \
+    "netbsd 386 lf-netbsd-386.tar.gz lf" \
+    "netbsd amd64 lf-netbsd-amd64.tar.gz lf" \
+    "netbsd arm lf-netbsd-arm.tar.gz lf" \
+    "openbsd 386 lf-openbsd-386.tar.gz lf" \
+    "openbsd amd64 lf-openbsd-amd64.tar.gz lf" \
+    "openbsd arm lf-openbsd-arm.tar.gz lf" \
+    "openbsd arm64 lf-openbsd-arm64.tar.gz lf" \
+    "solaris amd64 lf-solaris-amd64.tar.gz lf" \
+    "windows 386 lf-windows-386.zip lf.exe" \
+    "windows amd64 lf-windows-amd64.zip lf.exe"\
+    # Unsupported
+    # "aix ppc64 lf-aix-ppc64.tar.gz lf" \
+    # "android 386 lf-android-386.tar.gz lf" \
+    # "android amd64 lf-android-amd64.tar.gz lf" \
+    # "android arm lf-android-arm.tar.gz lf" \
+    # "darwin arm64 lf-darwin-arm64.tar.gz lf" \
+    # "js wasm lf-js-wasm.tar.gz lf" \
+    # "plan9 386 lf-plan9-386.tar.gz lf" \
+    # "plan9 amd64 lf-plan9-amd64.tar.gz lf" \
+    # "plan9 arm lf-plan9-arm.tar.gz lf" \
+do 
```

Comment:
I think it would be nice to `echo` some kind of heading here.

Also I was wondering what your opinion on [grouping log messages](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines) would be. I think grouping messages for a particular platform is nice, the only downside is that groups appear collapsed by default and if something fails then you have to open them to see what happened.

Reply:
> I think it would be nice to echo some kind of heading here.

I'll look at what you did and keep thinking about it. Currently, the script essentially outputs paragraphs separated by blank lines, which is not ideal.

> Also I was wondering what your opinion on [grouping log messages](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines) would be.

That's an interesting feature. In this case, however, I think it would be needlessly confusing for people who run `xbuild.sh` from the command-line.

Reply:
I added some headings and changed the output format to be a bit more dense.